requirements.txt

```diff
@@ -65,3 +65,7 @@ urllib3==1.25.10
 wcwidth==0.2.5
 Werkzeug==1.0.1
 zipp==3.1.0
+
+fastai~=2.0.8
+fastai2~=0.0.30
```

Comment:
Two versions of fastai?

Reply:
I checked and the fastai2 is not needed. Wierd behavior though